integration/oasis/osrmconnector/osrmrequest.go

```diff
@@ -0,0 +1,25 @@
+package osrmconnector
+
+// osrm request type
+const (
+	OSRMROUTE = iota
```

Comment:
Suggest to use camerl-case for const as convention in `Golang`.
https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const

Reply:
Will do.